setup.cfg

```diff
@@ -30,5 +30,22 @@ python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*
 console_scripts =
     pyupgrade = pyupgrade:main
 
+[options.package_data]
+pyupgrade =
+    py.typed
```

Comment:
Not sure if anyone is importing pyupgrade but it doesn't hurt to provide the stubs

Reply:
we can't we're a single file and not a package

Reply:
:mindblown: